hw/ip/entropy_src/doc/_index.md

```diff
@@ -112,8 +112,19 @@ This block will compress the bits such that the entropy bits/physical bits, or m
 The compression operation, by default, will compress every 2048 tested bits into 384 full-entropy bits.
 
 The hardware conditioning can also be bypassed and replaced in normal operation with a firmware-defined conditioning algorithm.
-This firmware conditioning algorithm, can be disabled on boot for security purposes.
-The exact mechanism for this functionality will be described in a future update to this document (TBD).
+This firmware conditioning algorithm can be disabled on boot for security purposes.
+
+The firmware override function has the capability to completely override the hardware health tests and the conditioner paths.
+In the case of health tests, firwmare can turn off one or all of the health tests and perform the tests in firmware.
+A data path is provided in the hardware such that the inbound entropy can be trapped in the pre-conditioner FIFO.
+Once a pre-determined threshold of entropy has been reached in this FIFO, the firmware can then read the entropy bits out of the FIFO.
+The exact mechanism for this functionality starts with setting the `FW_OV_MODE` field in the {{< regref "FW_OV_CONTROL" >}} register.
+This will enable firmware to monitor post-health test entropy bits by reading from the {{< regref "FW_OV_RD_DATA" >}} register.
+At this point, firmware can do additional health checks on the entropy.
+Optionally, firmware can do the conditioning function, assuming the the hardware is configured to bypass the conditioner block.
+Once firmware has processed the entropy,  it can then write the results back into the  {{< regref "FW_OV_WR_DATA" >}} register (pre-conditioner FIFO).
+The `FW_OV_ENTROPY_INSERT` in the {{< regref "FW_OV_CONTROL" >}} register will enable inserting entropy bits back into the entropy flow.
+The firmware override control bits will be set such that the new entropy will resume normal flow operation.
```

Comment:
Great start.  This essentially describes how to do this without conditioning.   Can you add a second paragraph that outlines how to use the FW_OV_SHA3_START register to do hardware based conditioning?

Reply:
I can. I was considering saying read the hjson for this, but text can be added as well.